---
layout: page
permalink: /publications/
title: Publications
description: 
nav_order: 1
---

### [](#header-3)Preprint

* Iida N, Okada A, Kobayashi Y, Chiba K, Yatabe Y, Shiraishi Y.
[Systematically developing a registry of splice-site creating variants utilizing massive publicly available transcriptome sequence data](https://www.biorxiv.org/content/10.1101/2024.02.21.581470v1).
bioRxiv, doi: https://doi.org/10.1101/2024.02.21.581470.
[Portal site](https://sscvdb.io/).

* Shiraishi Y, Chiba K, Okada A.
[SF3B1ness score: screening SF3B1 mutation status from over 60,000 transcriptomes based on a machine learning approach](https://www.biorxiv.org/content/10.1101/572834v1).
bioRxiv, doi: https://doi.org/10.1101/572834.
[Software](https://github.com/friend1ws/SF3B1ness).

### [](#header-3)Peer reviewed

The only peer-reviewed journals to which I have contributed as a first, co-first or corresponding author are shown.
The more comprehensive list may be found [here](https://www.ncbi.nlm.nih.gov/pubmed/?term=Yuichi+Shiraishi).

* Nakamura W, Hirata M, Oda S, Chiba K, Okada A, Mateos RN, Sugawa M, Iida N, Ushiama M, Tanabe N, Sakamoto H, Sekine S, Hirasawa A, Kawai Y, Tokunaga K; NCBN Controls WGS Consortium; Tsujimoto SI, Shiba N, Ito S, Yoshida T, Shiraishi Y.
[Assessing the efficacy of target adaptive sampling long-read sequencing through hereditary cancer patient genomes](https://www.nature.com/articles/s41525-024-00394-z).
NPJ Genom Med. 2024 Feb 17;9(1):11.

* Shiraishi Y\*, Koya J, Chiba K, Okada A, Arai Y, Saito Y, Shibata T, Kataoka K.
[Precise characterization of somatic complex structural variations from tumor/control paired long-read sequencing data with nanomonsv](https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkad526/7201946).
Nucleic Acids Res. 2023 Jun 20;gkad526.
[Software](https://github.com/friend1ws/nanomonsv).

* Shiraishi Y\*, Okada A\* et al.
[Systematic identification of intron retention associated variants from massive publicly available transcriptome sequencing data](https://www.nature.com/articles/s41467-022-32887-9).
Nat Commun. 2022 Sep 29;13(1):5357.
[Portal site](https://iravdb.io/).

* PCAWG Transcriptome Core Group et al. 
[Genomic basis for RNA alterations in cancer](https://www.nature.com/articles/s41586-020-1970-0).
Nature. 2020 Feb;578(7793):129-136.

* Shiraishi Y, Kataoka K, Chiba K, Okada A, Kogure Y, Tanaka H, Ogawa S, Miyano S.
[A comprehensive characterization of cis-acting splicing-associated variants in human cancer](https://genome.cshlp.org/content/early/2018/07/16/gr.231951.117).
Genome Res. 2018 Aug;28(8):1111-1125.

* Okada A, Chiba K, Tanaka H, Miyano S, Shiraishi Y. 
[A framework for generating interactive reports for cancer genome analysis](https://joss.theoj.org/papers/10.21105/joss.00457).
Journal of Open Source Software. 2017 2(20),457, doi:10.21105/joss.00457.
[Software](https://github.com/Genomon-Project/paplot).

* Kataoka K\*, Shiraishi Y\*, Takeda Y\* et al.
[Aberrant PD-L1 expression through 3'-UTR disruption in multiple cancers](https://www.nature.com/articles/nature18294).
Nature. 2016 Jun 16;534(7607):402-6.

* Shiraishi Y, Tremmel G, Miyano S, Stephens M. 
[A Simple Model-Based Approach to Inferring and Visualizing Cancer Mutation Signatures](https://doi.org/10.1371/journal.pgen.1005657).
PLoS Genet. 2015 Dec 2;11(12):e1005657.
[Software](https://github.com/friend1ws/pmsignature).

* Kataoka K\*, Nagata Y\*, Kitanaka A\*, Shiraishi Y\*, Shimamura T\*, Yasunaga J\*, Totoki Y\* et al.
[Integrated molecular analysis of adult T cell leukemia/lymphoma](https://www.nature.com/articles/ng.3415).
Nat Genet. 2015 Nov;47(11):1304-15.

* Chiba K\*, Shiraishi Y\*, Nagata Y, Yoshida K, Imoto S, Ogawa S, Miyano S. 
[Genomon ITDetector: a tool for somatic internal tandem duplication detection from cancer genome sequencing data](https://academic.oup.com/bioinformatics/article/31/1/116/2365706). 
Bioinformatics. 2015 Jan 1;31(1):116-8.
[Software](https://github.com/ken0-1n/Genomon-ITDetector).

* Shiraishi Y, Fujimoto A, Furuta M et al. 
[Integrated analysis of whole genome and transcriptome sequencing reveals diverse transcriptomic aberrations driven by somatic genomic changes in liver cancers](https://doi.org/10.1371/journal.pone.0114263). 
PLoS One. 2014 Dec 19;9(12):e114263.

* Sato Y\*, Yoshizato T\*, Shiraishi Y\*, Maekawa S\*, Okuno Y\* et al.
[Integrated molecular analysis of clear-cell renal cell carcinoma](https://www.nature.com/articles/ng.2699). 
Nat Genet. 2013 Aug;45(8):860-7.

* Shiraishi Y, Sato Y, Chiba K, Okuno Y, Nagata Y, Yoshida K, Shiba N, Hayashi Y, Kume H, Homma Y, Sanada M, Ogawa S, Miyano S. 
[An empirical Bayesian framework for somatic mutation detection from cancer genome sequencing data](https://academic.oup.com/nar/article/41/7/e89/1073733).
Nucleic Acids Res. 2013 Apr;41(7):e89.
[Software](https://github.com/friend1ws/EBCall). 

* Yoshida K\*, Sanada M\*, Shiraishi Y\*, Nowak D\*, Nagata Y\* et al.
[Frequent pathway mutations of splicing machinery in myelodysplasia](https://www.nature.com/articles/nature10496). 
Nature. 2011 Sep 11;478(7367):64-9.

* Shiraishi Y, Okada-Hatakeyama M, Miyano S. 
[A rank-based statistical test for measuring synergistic effects between two gene sets](https://academic.oup.com/bioinformatics/article/27/17/2399/223292). 
Bioinformatics. 2011 Sep 1;27(17):2399-405. 
[Software](https://github.com/friend1ws/rankSynergy).

* Shiraishi Y, Kimura S, Okada M. 
[Inferring cluster-based networks from differently stimulated multiple time-course gene expression data](https://academic.oup.com/bioinformatics/article/26/8/1073/208191). 
Bioinformatics. 2010 Apr 15;26(8):1073-81.
