<!doctype html>
<html lang="en-US">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">

<!-- Begin Jekyll SEO tag v2.3.0 -->
<title>Yuichi Shiraishi, Ph.D. | Assistant Professor&lt;/br&gt;Laboratory of DNA Information Analysis, Human Genome Center, The Institute of Medical Science, The University of Tokyo&lt;/br&gt;friend1ws [at] gmail [dot] com</title>
<meta property="og:title" content="Yuichi Shiraishi, Ph.D." />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Assistant Professor&lt;/br&gt;Laboratory of DNA Information Analysis, Human Genome Center, The Institute of Medical Science, The University of Tokyo&lt;/br&gt;friend1ws [at] gmail [dot] com" />
<meta property="og:description" content="Assistant Professor&lt;/br&gt;Laboratory of DNA Information Analysis, Human Genome Center, The Institute of Medical Science, The University of Tokyo&lt;/br&gt;friend1ws [at] gmail [dot] com" />
<link rel="canonical" href="http://localhost:4001/" />
<meta property="og:url" content="http://localhost:4001/" />
<meta property="og:site_name" content="Yuichi Shiraishi, Ph.D." />
<script type="application/ld+json">
{"name":"Yuichi Shiraishi, Ph.D.","description":"Assistant Professor&lt;/br&gt;Laboratory of DNA Information Analysis, Human Genome Center, The Institute of Medical Science, The University of Tokyo&lt;/br&gt;friend1ws [at] gmail [dot] com","author":null,"@type":"WebSite","url":"http://localhost:4001/","publisher":null,"image":null,"headline":"Yuichi Shiraishi, Ph.D.","dateModified":null,"datePublished":null,"sameAs":null,"mainEntityOfPage":null,"@context":"http://schema.org"}</script>
<!-- End Jekyll SEO tag -->


    <link rel="stylesheet" href="/assets/css/style.css?v=">
    <meta name="viewport" content="width=device-width">
    <!--[if lt IE 9]>
    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
  </head>
  <body>
    <div class="wrapper">
      <header>
        <img src="myphoto.jpg" alt="photo" width="250pt"> </td>
        <a href=""> <h2>Yuichi Shiraishi, Ph.D.</h2> </a>
        <p>Assistant Professor</br>Laboratory of DNA Information Analysis, Human Genome Center, The Institute of Medical Science, The University of Tokyo</br>friend1ws [at] gmail [dot] com</p>

        

        

        
          <ul>
            <li><a href="">Download <strong>ZIP File</strong></a></li>
            <li><a href="">Download <strong>TAR Ball</strong></a></li>
            <li><a href="">View On <strong>GitHub</strong></a></li>
          </ul>
        
      </header>
      <section>

      <h2 id="research-interest"><a href="#header-2"></a>Research Interest</h2>

<p>I have been working on developing statistical methods and analytical platforms for cancer genome sequencing data.</p>

<h3 id="variant-detection"><a href="#header-3"></a>Variant Detection</h3>

<p>Thanks to the advances in high-throughput sequence technologies,
genome-wide identification of genomic and transcriptomic changes is
now possible. However, due to ambiguities of short read alignment and
sequence errors, several types of genomic and transcriptome changes
(such as structural variations, gene fusions, complex indels and so
on) is stilll challenging issue. Furthermore, identification of
several omics data such as genomic variants influencing on
transcriptome changes (such as gene expression, splicing alteration
and so on) is still in imature phases.</p>

<p>I’ve been working on develping
statistical methods for identifying variety of genomic and
transcriptomic changes including but not limited to 
somatic mutations (<a href="https://www.ncbi.nlm.nih.gov/pubmed/23471004">Shiraishi et al., Nucleic Acids Research, 2013</a>),
gene fusions (<a href="https://www.ncbi.nlm.nih.gov/pubmed/25526364">Shiraishi et al., PLoS One, 2014</a>),
structural variations (<a href="https://github.com/Genomon-Project/GenomonSV">GenomonSV</a>).</p>

<p>Currently, I’m especially focusing on genomic variants affecting
splicing changes (<a href="https://www.biorxiv.org/content/early/2017/09/28/162560">Shiraishi et al., bioRxiv, 2017</a>).</p>

<h3 id="genome-analysis-platform"><a href="#header-3"></a>Genome Analysis Platform</h3>

<p>I have worked on developing cancer genome analytical pipeline, which
we call <a href="https://genomon-project.github.io/GenomonPagesR/">Genomon</a> since 2011, and have contributed to detections of a number of
novel cancer drivers in 
myelodysplasia (<a href="https://www.ncbi.nlm.nih.gov/pubmed/21909114">Yoshida, Sanada, Shiraishi et al., Nature, 2011</a>),
kidney cancers (<a href="https://www.ncbi.nlm.nih.gov/pubmed/23797736">Sato, Yoshizato, Shiraishi et al., Nature Genetics, 2013</a>),
adult T cell leukemia/lymphoma (<a href="https://www.ncbi.nlm.nih.gov/pubmed/26437031">Kataoka K, Nagata Y, Kitanaka A, Shiraishi Y et al., Nature Genetics, 2015</a>) and so on,
and common aberration of an immune checkpoint gene in multiple cancers 
(<a href="https://www.nature.com/articles/nature18294">Kataoka, Shiraishi et al., Nature, 2016</a>) through pan-cancer analysis.</p>

<p>Now, the cancer genome analysis
community is rapidly shifting to use cloud computing, and I’m very
interested in deploying our analytical pipeline on the cloud
environment with additional improvements.</p>

<h3 id="mutation-signature"><a href="#header-3"></a>Mutation Signature</h3>

<p>Cells throughout our body acquire somatic mutations caused by DNA replication errors or exposure to environmental mutagens such as tobacco smoking and some of these mutations can cause cancer.
Different cancer types, and even different instances of the same cancer types, can show different distinctive patterns of somatic mutations. 
For example, C &gt; A mutations are abundant in lung caners whereas C &gt; T and CC &gt; TT mutations are frequent in skin cancers. 
Thanks to the advances in high-throuput sequencing technologies, massive.
These distinctive patterns have become known as “mutation signatures”. 
Each mutation signature may be associated with a specific kind of carcinogen, such as tobacco smoke or ultraviolet light. Identifying mutation signatures therefore has the potential to identify new carcinogens, and yield new insights into the mechanisms and causes of cancer, In this paper, we introduce new statistical tools for tackling this important problem. These tools provide more robust and interpretable mutation signatures compared to previous approaches, as we demonstrate by applying them to large-scale cancer genomic data.</p>

<h2 id="selected-publications"><a href="#header-2"></a>Selected Publications</h2>

<h3 id="preprint"><a href="#header-3"></a>Preprint</h3>

<ul>
  <li>Okada A, Chiba K, Tanaka H, Miyano S, Shiraishi Y. 
<a href="https://www.biorxiv.org/content/early/2017/10/02/194035">A framework for generating interactive reports for cancer genome analysis</a>.
bioRxiv, doi: https://doi.org/10.1101/194035. 
<a href="https://github.com/Genomon-Project/paplot">Software</a>.</li>
  <li>Shiraishi Y, Kataoka K, Chiba K, Okada A, Kogure Y, Tanaka H, Ogawa S, Miyano S.
<a href="https://www.biorxiv.org/content/early/2017/09/28/162560">A comprehensive characterization of cis-acting splicing-associated variants in human cancer</a>. 
bioRxiv, doi: https://doi.org/10.1101/162560. 
<a href="https://github.com/friend1ws/SAVNet">Software</a>.</li>
</ul>

<h3 id="peer-reviewed"><a href="#header-3"></a>Peer reviewed</h3>

<p>Only peer reviewed journals to which I have contributed as a first, co-first or corresponding author are shown.
More comprehensive list may be found in <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Yuichi+Shiraishi">here</a>.</p>

<ul>
  <li>Kataoka K*, Shiraishi Y*, Takeda Y* et al.
<a href="https://www.nature.com/articles/nature18294">Aberrant PD-L1 expression through 3’-UTR disruption in multiple cancers</a>.
Nature. 2016 Jun 16;534(7607):402-6.</li>
  <li>Shiraishi Y, Tremmel G, Miyano S, Stephens M. 
<a href="https://doi.org/10.1371/journal.pgen.1005657">A Simple Model-Based Approach to Inferring and Visualizing Cancer Mutation Signatures</a>.
PLoS Genet. 2015 Dec 2;11(12):e1005657.
<a href="https://github.com/friend1ws/pmsignature">Software</a>.</li>
  <li>Kataoka K+, Nagata Y*, Kitanaka A*, Shiraishi Y*, Shimamura T*, Yasunaga J*, Totoki Y* et al.
<a href="https://www.nature.com/articles/ng.3415">Integrated molecular analysis of adult T cell leukemia/lymphoma</a>.
Nat Genet. 2015 Nov;47(11):1304-15.</li>
  <li>Chiba K*, Shiraishi Y*, Nagata Y, Yoshida K, Imoto S, Ogawa S, Miyano S. 
<a href="https://academic.oup.com/bioinformatics/article/31/1/116/2365706">Genomon ITDetector: a tool for somatic internal tandem duplication detection from cancer genome sequencing data</a>. 
Bioinformatics. 2015 Jan 1;31(1):116-8.
<a href="https://github.com/ken0-1n/Genomon-ITDetector">Software</a>.</li>
  <li>Shiraishi Y, Fujimoto A, Furuta M et al. 
<a href="https://doi.org/10.1371/journal.pone.0114263">Integrated analysis of whole genome and transcriptome sequencing reveals diverse transcriptomic aberrations driven by somatic genomic changes in liver cancers</a>. 
PLoS One. 2014 Dec 19;9(12):e114263.</li>
  <li>Sato Y*, Yoshizato T*, Shiraishi Y*, Maekawa S*, Okuno Y* et al.
<a href="https://www.nature.com/articles/ng.2699">Integrated molecular analysis of clear-cell renal cell carcinoma</a>. 
Nat Genet. 2013 Aug;45(8):860-7.</li>
  <li>Shiraishi Y, Sato Y, Chiba K, Okuno Y, Nagata Y, Yoshida K, Shiba N, Hayashi Y, Kume H, Homma Y, Sanada M, Ogawa S, Miyano S. 
<a href="https://academic.oup.com/nar/article/41/7/e89/1073733">An empirical Bayesian framework for somatic mutation detection from cancer genome sequencing data</a>.
Nucleic Acids Res. 2013 Apr;41(7):e89.
<a href="https://github.com/friend1ws/EBCall">Software</a>.</li>
  <li>Yoshida K*, Sanada M*, Shiraishi Y*, Nowak D*, Nagata Y* et al.
<a href="https://www.nature.com/articles/nature10496">Frequent pathway mutations of splicing machinery in myelodysplasia</a>. 
Nature. 2011 Sep 11;478(7367):64-9.</li>
  <li>Shiraishi Y, Okada-Hatakeyama M, Miyano S. 
<a href="https://academic.oup.com/bioinformatics/article/27/17/2399/223292">A rank-based statistical test for measuring synergistic effects between two gene sets</a>. 
Bioinformatics. 2011 Sep 1;27(17):2399-405. 
<a href="https://github.com/friend1ws/rankSynergy">Software</a>.</li>
  <li>Shiraishi Y, Kimura S, Okada M. 
<a href="https://academic.oup.com/bioinformatics/article/26/8/1073/208191">Inferring cluster-based networks from differently stimulated multiple time-course gene expression data</a>. 
Bioinformatics. 2010 Apr 15;26(8):1073-81.</li>
</ul>



      </section>
      <footer>
        
        <p><small>Hosted on GitHub Pages &mdash; Theme by <a href="https://github.com/orderedlist">orderedlist</a></small></p>
      </footer>
    </div>
    <script src="/assets/js/scale.fix.js"></script>


  
  </body>
</html>
